add ServiceCard component (#200)
patternfly/react-component-groups: packages/module/patternfly-docs/content/extensions/component-groups/examples/ServiceCard/ServiceCard.md

---
section: extensions
subsection: Component groups
id: Service card
source: react
propComponents: ['ServiceCard']
sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/examples/ServiceCard/ServiceCard.md
---

import ServiceCard from "@patternfly/react-component-groups/dist/dynamic/ServiceCard";
import { EllipsisVIcon } from '@patternfly/react-icons';
import contentHeaderIcon from '../../assets/icons/content-header-icon.svg'

A **Service Card** component allows

## Examples

### Service Card

```js file="./ServiceCardExample.tsx"

```

### Service Card with Gallery example

```js file="./ServiceCardGalleryExample.tsx"

> ```

Reviewer: @InsaneZein can you please add some texts to the docs? It would also be nice to create an example with the custom footer. Let me know if you need help with the texts.

---
section: extensions
subsection: Component groups
id: Service card
source: react
propComponents: ['ServiceCard']
sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/examples/ServiceCard/ServiceCard.md
---

import ServiceCard from "@patternfly/react-component-groups/dist/dynamic/ServiceCard";
import { EllipsisVIcon } from '@patternfly/react-icons';
import contentHeaderIcon from '../../assets/icons/content-header-icon.svg'

> A **Service Card** component displays a card with an icon, title, description, and an optional customized footer

Reviewer: ```suggestion
The **service card** component displays a card representing a service with an icon, title, description, and an optional customized footer
```

## Examples

### Service Card

This shows a basic service card with an `icon`, `title`, `description`, and optional footer passed in.

```js file="./ServiceCardExample.tsx"

```

### Service Card with Gallery example

> This shows how cards will look side by side in a `Gallery` component

Reviewer: ```suggestion
This shows how cards can look side by side in a [gallery layout]((/layouts/gallery))
```
